sdk/devcenter/Azure.Developer.DevCenter/tests/Azure.Developer.DevCenter.Tests.csproj

```diff
@@ -21,8 +21,7 @@
   </ItemGroup>
   
   <ItemGroup>
-    <Folder Include="SessionRecords\EnvironmentCreateTests\" />
-    <Folder Include="SessionRecords\DevBoxCreateTests\" />
+    <Folder Include="SessionRecords\DevCenterClientTests\" />
```

Comment:
I don't think you need to add this to the project. Do you see any errors if you don't include it to the project.

Reply:
I also see the project taking dependency on Microsoft.Identity.Client SDK. Why is it needed?

Reply:
Removed the records folder from the csproj. `Microsoft.Identity.Client` package is used in `DevCenterTestUserCredential` class to acquire an access token.

Reply:
@olegoid You don't need to define `DevCenterTestUserCredential` class. You can use [TestEnvironment.Credential](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core.TestFramework/src/TestEnvironment.cs#L195).

Here are some usages:
* https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/synapse/Azure.Analytics.Synapse.Spark/tests/SparkSessionClientLiveTests.cs#L34
* UsernamePasswordCredential usage - https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/purview/Azure.Analytics.Purview.Workflows/tests/WorkflowsClientTestEnvironment.cs#L20

Reply:
Switched to `TestEnvironement.Credential` but since most of our tests are playback only the credential will be mocked most of the time.